apollo-configservice/src/main/resources/application.yml

```diff
@@ -38,22 +38,22 @@ logging:
 eureka:
   instance:
     hostname: ${hostname:localhost}
-    preferIpAddress: true
+    prefer-ip-address: true
     status-page-url-path: /info
     health-check-url-path: /health
   server:
-    peerEurekaNodesUpdateIntervalMs: 60000
-    enableSelfPreservation: false
+    peer-eureka-nodes-update-interval-ms: 60000
+    enable-self-preservation: false
   client:
-    serviceUrl:
+    service-url:
       # This setting will be overridden by eureka.service.url setting from ApolloConfigDB.ServerConfig or System Property
       # see com.ctrip.framework.apollo.biz.eureka.ApolloEurekaClientConfig
       defaultZone: http://${eureka.instance.hostname}:8080/eureka/
     healthcheck:
       enabled: true
-    eurekaServiceUrlPollIntervalSeconds: 60
+    eureka-service-url-poll-interval-seconds: 60
     fetch-registry: false
-    registerWithEureka: false
+    registe-with-eureka: false
```

Comment:
I think the major effort of this pull request is to verify whether the configurations still work after renaming. e.g. the key name is mis-spelled.

Reply:
Yes, I have tested and confirmed that the eureka and LDAP configuration is effective